### ISSUE TITLE
SRM not creating proper subdirectory on fresh install

### ISSUE THREAD

**Issue by klepp0906** [doZennn/steam-rom-manager#600]
TLDR i wiped/re-installed steam.  Ran SRM 2.4.17 with the same set of parsers ive been using for years now.  Nothing was taking imagewise (local images).  All the shortcuts were there fine, but sans artwork.

logs gave me a whole lot of this.
![image](https://github.com/SteamGridDB/steam-rom-manager/assets/10392920/53906c41-7a6f-469c-8fd8-e342b9ed80af)

turns out if i went into ``C:\Program Files (x86)\Steam\userdata\157320782\config`` and created the ``grid`` sub directory manually then it  would populate the images fine.

before even getting to that point I had assumed somehow, some kind of permissions issue and so I ran SRM as admin but still no bueno.

**Comment by cbartondock** [comment]
Fixed in master.
